build/build.yml

```diff
@@ -0,0 +1,116 @@
+pool:
+  name: Hosted macOS
+  demands:
+  - npm
+  - node.js
+
+steps:
+- task: ArchiveFiles@1
```

Comment:
ok in this instance these _actually_ are useless. At least in ads it was capturing compiled files. Here you can see the same code by just looking at the commit.

Reply:
Yeah - we should do a cleanup of these since there's a lot of useless/duplicate tasks that have built up over time. Was planning on doing that in a separate PR though as committer stuff later on.